### DISCUSSION TITLE
Receiving a transmission on a different frequency?!

### DISCUSSION THREAD

**Discussion by homeautomaton** [merbanan/rtl_433#2449] [Q&A] [answered]
I'm pretty new to SDR in general, and using some cheap usb SDR from Aliexpress. I'm surprised, though, with the device tuned to 315MHz I am seeing, along with plenty of other plausible transmissions in the proper band, a device which I am 100% sure is transmitting on 433.92MHz.

I have a second SDR always running on 433M, and it definitely receives what I know 100% sure is a LaCrosse-TX141THBv2. The sensor is about 3 meters from the 315M receiver, if that's a factor. On both frequencies, the receptions are simultaneous, and if I take the batteries out of the sensor, they both stop.

I'm using all of the default settings for the radio, so I don't expect a wide enough bandwidth for even a poorly tuned 433M transmitter to wind up close enough to 315M. And I don't think this would be a matter of harmonics(?)

Maybe something about my receiver is a clue?
```
Found Rafael Micro R820T tuner
Exact sample rate is: 250000.000414 Hz
[R82XX] PLL not locked!
Sample rate set to 250000 S/s.
Tuner gain set to Auto.
Tuned to 315.000MHz.
baseband_demod_FM: low pass filter for 250000 Hz at cutoff 25000 Hz, 40.0 us
```

**Comment by zuckschwerdt** [comment, marked as answer]
This should not happen regularly, but:
The RTL-SDR has no effective band filter.
The LaCrosse-TX141THBv2 is OOK PWM, which is easy to pick up and recover.
Strong signals will induce spectral noise in the antenna, which can be seen as OOK signal.

**Reply by homeautomaton** [reply]
Thanks! I thought it might be some RF/physics phenomenon. It was a little crazy for me, because I'd modified the electronics of this particular TX141THB, using a transmitter module I was only 99% sure was the correct frequency. As I'm sure you are aware, the circuit boards for said modules don't always clearly indicate their frequency, but the crystal frequency matched another 433MHz device...

Why modify the TX141THB in the first place? It was purchased used, along with the color weather display, for $1 at a garage sale. Corroded batteries and a rusted antenna were obvious problems, but even cleaned up and with a replaced antenna, it wasn't transmitting. An oscilloscope showed the on-board transmitter module did have pulses on its input pin, so it seemed logical to piggy-back a working transmitter.

Days later I discovered the bleed-over onto the 315M band, and it made me question what module I'd put in, and whether the original on-board transmitter might've finally awakened, even with no antenna...

With your explanation, I can put my concerns to rest. Thanks.

**Reply by zuckschwerdt** [reply]
Could also be the module then. We've seen some pretty bad TX modules. There might be harmonics and noise. Esp. if it just "clocks a pin" to transmit :)

**Reply by homeautomaton** [reply]
Interesting. I'm not sure how much a picture might tell you, but here's the module (bought on eBay a long time back):

![image](https://user-images.githubusercontent.com/48721345/227551714-4baca5e4-a687-4659-b1af-6fc2b8e44b34.png)
